app/helpers/projects_helper.rb

```diff
@@ -0,0 +1,31 @@
+module ProjectsHelper
+  def status_label(status, id, tag = :span)
+    label_class = label_class_lookup(status)
+    id = 'job-status-span-' + id.to_s
+
+    content_tag tag, class: %I[status-label label #{label_class}], id: id do
+      status
+    end
+  end
+
+  def thumbnails(project_dir)
+    imgs = Dir.glob(project_dir + '/thumbnails/*.png')
+    imgs&.sort
```

Comment:
It would improve readability here to just do one line:

```
def thumbnails(project_dir)
  Dir.glob(project_dir + '/thumbnails/*.png').sort
end
```

Also you don't need to use the `&.` because Dir.glob will always return an array.

Reply:
Done.